fixtures/redshift_schema_edl.sql

```diff
@@ -1,12 +1,3 @@
-CREATE SCHEMA IF NOT EXISTS {redshift_schema_edl_external};
-
-CREATE TABLE IF NOT EXISTS {redshift_schema_edl_external}.student_academic_plan_data (
-    student_id VARCHAR NOT NULL,
-    academic_career_cd VARCHAR,
-    academic_program_status_cd VARCHAR,
-    academic_plan_type_cd VARCHAR
-);
-
```

Comment:
@lyttam new tables look good, but we probably don't want to lose this fixture table for test purposes?

Reply:
@pauline2k Ah, it didn't look like it was doing anything but apparently it is! Amended.